packages/shared/src/hooks/notifications/useBookmarkReminder.ts

```diff
@@ -117,19 +121,34 @@ export const useBookmarkReminder = (): UseBookmarkReminder => {
         throw new Error('Invalid preference');
       }
 
+      logEvent(
+        postLogEvent(LogEvent.SetBookmarkReminder, post, {
+          extra: { remind_in: preference },
+        }),
+      );
+
       return onSetBookmarkReminder({
         ...props,
         remindAt: getRemindAt(now, preference),
       });
     },
-    [onSetBookmarkReminder],
+
+    // logEvent is known to be unstable
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Better to just put it on? It is not as bad as in `useEffect`.

Reply:
Right. Yeah, we can do that.